modules/cleaning-up-after-deleting-users.adoc

```diff
@@ -0,0 +1,67 @@
+:_module-type: PROCEDURE
+
+[id='cleaning-up-after-deleting-users_{context}']
+= Cleaning up after deleting users
+
+[role='_abstract']
+ifndef::self-managed[]
+After removing a user's access to {productname-long} or Jupyter, you must also delete their associated configuration files from {openshift-platform}.
+endif::[]
+ifdef::self-managed[]
+After removing a user's access to {productname-long} or Jupyter, you must also delete their associated configuration files from OpenShift Container Platform.
```

Comment:
We probably also want to use  {openshift-platform} for OpenShift Container Platform and let the upstream/downstream and managed/self-managed conditionalization control what value appears in the output? I don't think we want it in plain text.

Reply:
done